src/components/shared/Tray/TrayComponents/SelectedService/customHooks/useShowSelectedServiceInfo.js

```diff
@@ -0,0 +1,87 @@
+import { useContext } from 'react';
+import { FetchDisruptionsContext, AutoCompleteContext, ModeContext } from 'globalState';
+// Import customHooks
+import useFilterLogic from 'customHooks/useFilterLogic';
+
+const useShowSelectedServiceInfo = () => {
+  const [{ isMapVisible }] = useContext(FetchDisruptionsContext);
```

Comment:
Double destructure...nice. 
Be careful not to overuse this in future as it can cause confusion.

For now, as it is one item, it is simple enough to understand.